ai-service/tests/test_llm_and_search.py

```
@@ -3,74 +1,79 @@
+    # Ensure db is empty
+    test_collection = get_collection()
+    if test_collection.peek()["ids"]:
+        test_collection.delete(ids=test_collection.peek()["ids"])
+    question = "This code does not exist."
+    question_embedding = embed_query(question)
+    results = query_chunks(
+        question_embedding,
+    )
+    # Should return an empty or placeholder result
+    docs = results.get("documents")
+    assert docs is not None and (docs[0] == [] or (docs[0] and docs[0][0] == ""))
```

Ruff (0.12.2)

Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)